### ISSUE TITLE
As a user, I would like to know that my Christmas tree permit in Shoshone National Forest is only for harvesting trees less than 20 feet tall 

### ISSUE THREAD

**Issue by sethalt** [nciinc/fs-intake-module#132]
## Notes
In Shoshone National Forest, only trees under 20' tall can be harvested. Trees larger than 20' can be harvested, but a permit needs to be obtained from the Forest Service office.

## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by sethalt** [comment]
@JaneZC The following text needs to change to explain to the permittee that while you can only get a permit online to cut a tree under 20' tall, you can still get a permit from the forest service office to cut trees larger than 20' tall.

![screen shot 2017-11-29 at 1 18 11 pm](https://user-images.githubusercontent.com/2347759/33394475-194d75b2-d508-11e7-80a2-7c04422fae39.png)

**Comment by JaneZC** [comment]
Suggested language below; @adsprinkle @MelissaBraxton @sethalt 

Question/Suggestion: This particular piece of content (ie. where to get a permit for trees over 20 feet) feels more like a helpful hint, rather than a rule (especially since the rules already say the permit is for trees 20 feet or less in height). Perhaps we can add this content to the Contact Us section?

If the content stays in "Rules for cutting your tree" try:

For trees over 20 feet tall, permits must be purchased in person at Shoshone Ranger District offices. 

(then hyperlink to Contact Us section)

If content goes in Contact Us say....

Permits to cut trees over 20 feet tall are not available online and can only be purchased in person at Shoshone Ranger District offices.

**Comment by sethalt** [comment]
@ASprinkle @MelissaBraxton @JaneZC 

I added the text that Jane suggested, and edited the line above to read: "The tree must be 20 feet or less if permit was purchased online," instead of "The tree must be 20 feet or less"

![screen shot 2017-11-30 at 1 52 05 pm](https://user-images.githubusercontent.com/2347759/33451790-aff30716-d5d5-11e7-9cd9-38bb94671122.png)

**Comment by ASprinkle** [comment]
I agree with the information there. The 20' foot descriptions look really good. I appreciate the link included.

**Comment by MelissaBraxton** [comment]
Looks good to me!
